source/renderer/app/themes/overrides/PopOverOverrides.scss

```diff
@@ -1,2 +1,5 @@
 .root {
+  :global .tippy-content {
```

Comment:
@DominikGuzei isn't there a better way to target the content and set up the line-height?

Reply:
@nikolaglumac good point, we could probably 1. add a sane default in react-polymorph and 2. make it configurable via theme variable 👍 I also saw that most of the `maxWidth` values are 350, so I will also add that as default

Reply:
Will you take care of that?